python/python/glide/async_commands/command_args.py

```diff
@@ -63,3 +63,18 @@ class ListDirection(Enum):
     """
     RIGHT: Represents the option that elements should be popped from or added to the right side of a list.
     """
+
+
+class FlushMode(Enum):
```

Comment:
add to __init__.py

Reply:
Could have a conflict with #1579 - please double check before merging

Reply:
I've merged https://github.com/aws/glide-for-redis/pull/1579. Please rebase and update `FlushMode(Enum)` to serve both commands.